### PR TITLE
Create alokik.json

### PR DESCRIPTION
**Pull request by OpAayush**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://alokik-here.github.io/devloper-alokik

### CODE REVIEW

**Comment by OpAayush** [comment]
Please merge 🧛🏻‍♂️

**Comment by WilliamDavidHarrison** [comment]
> Please merge 🧛🏻‍♂️

Please fix the JSON

**Comment by OpAayush** [comment]
What about now

File: domains/alokik.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+      "owner": {
+        "username": "alokik-here",
```
Comment (WilliamDavidHarrison) [inline domains/alokik.json:3]: Sorry, you can't register domains in other peoples names.

File: domains/alokik.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        "email": "aayushmagrawal@gmail.com"
+      },
+      "record": {
+        "CNAME": "alokik-here.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/alokik.json:7]: 404 and also please see the review comment above.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/alokik.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+	"owner": {
+		"username": "OpAayush",
+		"email": "aayushmagrawal@gmail.com",
+                "repo": "https://github.com/alokik-here/devloper-alokik
```
[the remark this answers is not in the document]
Comment (WilliamDavidHarrison) [reply domains/alokik.json]: Fixed

**Comment by OpAayush** [comment]
Bro he is my brother :/

File: domains/alokik.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+      "owner": {
+        "username": "OpAayush",
+        "email": "aayushmagrawal@gmail.com"
+      },
+      "record": {
+        "CNAME": "alokik-here.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/alokik.json:7]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by OpAayush** [comment]
The website is https://alokik-here.github.io/devloper-alokik

**Comment by OpAayush** [comment]
??

**Review by wdhdev — APPROVED** [review]

**Pull request merged**